app/javascript/components/users/SignInPage.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import MockCurrentUserProvider from '../tests/MockCurrentUserProvider';
+import SignInComponent, { } from './SignInComponent';
+import SignInPage, { } from './SignInPage';
```

Comment:
As a reminder, please avoid empty curly brace imports. For line 5, this would be better written as 

```typescript
import SignInComponent from './SignInComponent';
```

Reply:
Sounds good! We'll fix that